Sources/Commands/Workspace.swift

```diff
@@ -269,6 +271,71 @@ public class Workspace {
         return path
     }
 
+    /// Updates the current dependencies.
+    public func updateDependencies() throws {
+        let (rootManifest, _) = try loadDependencyManifests()
+        // Only create constraints based on root manifest for the update resolution.
+        let updateConstraints = computePackageConstraints(rootManifest: rootManifest, dependencies: [])
+        // Resolve the dependencies.
+        let updateResults = try resolveDependencies(constraints: updateConstraints)
+
+        for (specifier, version) in updateResults {
+            // If the specifier's version did not update then just continue.
+            if let currentDependency = dependencyMap[specifier],
+               let currentVersion = currentDependency.currentVersion, 
+               currentVersion == version {
+                continue
+            }
+            let container = try containerProvider.getContainer(for: specifier)
+            guard let tag = container.getTag(for: version) else {
```

Comment:
Let's keep a FIXME around here that we should ideally be able to get this information directly back from the provider rather than having to reopen and resolve.